lib/facter/util/file_helper.rb

```diff
@@ -9,14 +9,14 @@ class << self
         DEBUG_MESSAGE = 'File at: %s is not accessible.'
 
         def safe_read(path, default_return = '')
-          return File.read(path) if File.readable?(path)
+          return File.read(path, :encoding => 'UTF_8') if File.readable?(path)
```

Comment:
Nit better to use the encoding constant instead of a temporary string:

```suggestion
          return File.read(path, :encoding => Encoding::UTF_8) if File.readable?(path)
```